.github/workflows/ci.yaml

```diff
@@ -0,0 +1,68 @@
+name: "CI"
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master 
+  create:
+    tags:
+      - v*
```

Comment:
I am nitpicking now, but I think this might be more readable:

```yml
on:
  push: { branches: [master] }
  pull_request: { branches: [master] }
  create: { tags: [v*] }
```

Reply:
Fixed :)